gradle.properties

```diff
@@ -56,4 +56,6 @@ mod_group_id=com.moujounakki.archimedesfluids
 # The authors of the mod. This is a simple text string that is used for display purposes in the mod list.
 mod_authors=MouJouNakki
 # The description of the mod. This is a simple multiline text string that is used for display purposes in the mod list.
-mod_description=Overhauls how fluids flow.
\ No newline at end of file
+mod_description=Overhauls how fluids flow.
+# JSON Simple library dependency
+implementation 'com.googlecode.json-simple:json-simple:1.1.1'
```

Comment:
What is this used for?

Reply:
That is leftover from a config file I was attempting to integrate / still am. At the moment can be removed if need be